src/wallet/rpcwallet.cpp

```diff
@@ -1761,7 +1776,10 @@ UniValue listsinceblock(const JSONRPCRequest& request)
             "    \"comment\": \"...\",       (string) If a comment is associated with the transaction.\n"
             "    \"label\" : \"label\"       (string) A comment for the address/transaction, if any\n"
             "    \"to\": \"...\",            (string) If a comment to is associated with the transaction.\n"
-             "  ],\n"
+            "  ],\n"
+            "  \"removed\": [\n"
```

Comment:
Can we add something to note that transactions which were re-added are included here anyway, and may have, at that point, positive confirmations value in this array?